docs/blog/20230401-confused-Techie-PON.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "Meet PON (Pulsar Object Notation)"
+author: confused-Techie
+date: 2023-04-01
+category:
+  - dev
+tag:
+  - joke
+---
+
+The release of PON!
+
+<!-- more -->
+
+One day during discussions amoung the Pulsar dev team, we came to the topic of what file types to support for user's config files. While it was mostly a question of how support would go with allowing anything from JSON, JSON5, TOML, YAML, CSON, and anything else we coudl think of, in the end we couldn't agree on a solution.
+
+We kept finding downsides to every possibility, and concerns for support with so many. So we decided it only made sense, to take the true Atom approch, and create something new.
```

Comment:
maybe something like:

```suggestion
We kept finding downsides to every possibility, and concerns for support with so many. So we decided it only made sense, to take the true Atom approch, and reinvent the wheel with an in-house solution.
```

Reply:
Thank you, couldn't remember what you said previously, but was trying to quote your earlier comment here lol